### PR TITLE
Roll version for 1.10 development

### PR DESCRIPTION
**Pull request by trbehera**
Signed-off-by: Behera, Tushar Ranjan <tushar.ranjan.behera@intel.com>

### CODE REVIEW

**Review by KiranSukhavasi — APPROVED** [review]
Approved.

**Review by DukeDavis12 — APPROVED** [review]

**Review by prateekcr — DISMISSED** [review]

File: Jenkinsfile.yml [hunk 1]
```diff
@@ -16,12 +17,14 @@ node('ubuntu18.04-OnDemand'){
       sh 'git clone "${REPO_PRI}"'
       sh 'git clone "${REPO_SCT}"'
```
Comment (DukeDavis12) [inline Jenkinsfile.yml:18]: Missing Rollover in container/tomcat/conf/catalina.properties. Line 133

**Review by DukeDavis12 — CHANGES_REQUESTED** [review]
Missing version Rollover in container/tomcat/conf/catalina.properties. 
Line 133 
`application.version=1.9 `

File: Jenkinsfile.yml (hunk 1 above)
Comment (trbehera) [reply Jenkinsfile.yml:18]: Done.

**Review by DukeDavis12 — APPROVED** [review]

**Review by prateekcr — APPROVED** [review]

**Pull request merged**